### ISSUE TITLE
Suggest больше не работает

### ISSUE THREAD

**Issue by labtorie** [volga-volga/react-native-yamap#203]
Кажется, Яндекс решил поменять респонс геокодера, теперь там что-то вроде такого
` "uri": "ymapsbm1://geo?data=CgozMzk3ODcxOTQ1EnnQoNC+0YHRgdC40Y8sINCj0LTQvNGD0YDRgtGB0LrQsNGPINCg0LXRgdC/0YPQsdC70LjQutCwLCDQmNC20LXQstGB0LosINCm0LLQtdGC0L7Rh9C90LDRjyDRg9C70LjRhtCwLCA2LCDQv9C+0LTRitC10LfQtCAxIgoNSRFVQhWffWNC"`
Соответственно, координаты оттуда не достать, расшифровка BASE64 их не дает.

**Comment by DimitrenkoNikolay** [comment]
> Кажется, Яндекс решил поменять респонс геокодера, теперь там что-то вроде такого ` "uri": "ymapsbm1://geo?data=CgozMzk3ODcxOTQ1EnnQoNC+0YHRgdC40Y8sINCj0LTQvNGD0YDRgtGB0LrQsNGPINCg0LXRgdC/0YPQsdC70LjQutCwLCDQmNC20LXQstGB0LosINCm0LLQtdGC0L7Rh9C90LDRjyDRg9C70LjRhtCwLCA2LCDQv9C+0LTRitC10LfQtCAxIgoNSRFVQhWffWNC"` Соответственно, координаты оттуда не достать, расшифровка BASE64 их не дает.

а я сижу и не понимаю, почему вчера еще координаты были, а сегодня уже нет) ты использовал `suggestWithCoords` этот метод?

**Comment by labtorie** [comment]
@DimitrenkoNikolay да, suggestWithCoords. Там он работал за счет того, что из uri сплитом доставал координаты. 

Пока что экстренно обошел с `https://geocode-maps.yandex.ru/1.x?apikey=${getRandomKey()}&format=json&results=1&geocode=${item.name}`
